packages/dw-refresh-cache/package.json

```
@@ -0,0 +1,28 @@
+{
+	"name": "@dlw-digitalworkplace/dw-refresh-cache",
+	"version": "0.0.0",
```

Should this be 0.0.1 maybe?
```suggestion
	"version": "0.0.1",
```